### PR TITLE
[1LP][RFR] Update SSHResult

### PR DESCRIPTION
**Pull request by mshriver**
Make SSHResult a proper class with attributes, instead of just a namedtuple.

Add `command` attribute, set in SSHClient to the command string that was actually called.

Thanks to some brainstorming with @jamesooden

### CODE REVIEW

File: cfme/utils/ssh.py [hunk 1]
```diff
@@ -49,6 +51,10 @@ class SSHResult(namedtuple("SSHResult", ["rc", "output"])):
     But it still subclasses the original class therefore all old behaviour is kept. But you don't
     have to expand the tuple or pull the value out if you are checking only one of them.
     """
+    rc = attr.ib()
```
Comment (RonnyPfannschmidt) [inline cfme/utils/ssh.py]: **required** please rename the attributes to `return_code` and `command`

additionally set `repr=False` on `output` and reorder to `command, return_code, output`

that way we get a nicer repr that's readable on terminal

**Comment by dajoRH** [comment]
Lint report for commit eede88a004886f7e20864a6e147172453ad26081:

`cfme/tests/cli/test_appliance_update.py`:
- :red_circle: Line 90:58: **F821** *undefined name 'out'*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 469:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 502:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 504:101: **E501** *line too long (105 > 100 characters)*

`cfme/tests/intelligence/reports/test_validate_chargeback_report.py`:
- :red_circle: Line 205:72: **F821** *undefined name 'out'*
- :red_circle: Line 239:67: **F821** *undefined name 'out'*

`cfme/tests/ssui/test_ssui_dashboard.py`:
- :red_circle: Line 126:12: **F821** *undefined name 'result'*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b8546794f6608234acbf0a45c326926075556c26:

`cfme/tests/cli/test_appliance_update.py`:
- :red_circle: Line 90:58: **F821** *undefined name 'out'*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 469:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 502:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 504:101: **E501** *line too long (105 > 100 characters)*

`cfme/tests/intelligence/reports/test_validate_chargeback_report.py`:
- :red_circle: Line 205:72: **F821** *undefined name 'out'*
- :red_circle: Line 239:67: **F821** *undefined name 'out'*

`cfme/tests/ssui/test_ssui_dashboard.py`:
- :red_circle: Line 126:12: **F821** *undefined name 'result'*

`cfme/utils/appliance/__init__.py`:
- :red_circle: Line 1530:17: **F841** *local variable 'result' is assigned to but never used*
- :red_circle: Line 1559:9: **F841** *local variable 'result' is assigned to but never used*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 95753059c005fa761abc24f96687e72316cc5fe7:

`cfme/tests/cli/test_appliance_update.py`:
- :red_circle: Line 90:58: **F821** *undefined name 'out'*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 469:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 502:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 504:101: **E501** *line too long (105 > 100 characters)*

`cfme/tests/intelligence/reports/test_validate_chargeback_report.py`:
- :red_circle: Line 205:72: **F821** *undefined name 'out'*
- :red_circle: Line 239:67: **F821** *undefined name 'out'*

`cfme/tests/ssui/test_ssui_dashboard.py`:
- :red_circle: Line 126:12: **F821** *undefined name 'result'*

`cfme/utils/appliance/__init__.py`:
- :red_circle: Line 1530:17: **F841** *local variable 'result' is assigned to but never used*
- :red_circle: Line 1559:9: **F841** *local variable 'result' is assigned to but never used*

`scripts/fetch_jenkins_artifacts.py`:
- :red_circle: Line 32:9: **F841** *local variable 'result' is assigned to but never used*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 76023caa55b6cbeaedd819916f9cbfa677589bbb:

`cfme/tests/cli/test_appliance_update.py`:
- :red_circle: Line 90:58: **F821** *undefined name 'out'*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 469:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 502:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 504:101: **E501** *line too long (105 > 100 characters)*

`cfme/tests/intelligence/reports/test_validate_chargeback_report.py`:
- :red_circle: Line 205:72: **F821** *undefined name 'out'*
- :red_circle: Line 239:67: **F821** *undefined name 'out'*

`cfme/tests/ssui/test_ssui_dashboard.py`:
- :red_circle: Line 126:12: **F821** *undefined name 'result'*

`cfme/utils/appliance/__init__.py`:
- :red_circle: Line 1530:17: **F841** *local variable 'result' is assigned to but never used*
- :red_circle: Line 1559:9: **F841** *local variable 'result' is assigned to but never used*

`scripts/fetch_jenkins_artifacts.py`:
- :red_circle: Line 32:9: **F841** *local variable 'result' is assigned to but never used*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 531162b8d4e37758aba03ca1bf2e420ce1b17d18:

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 469:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 502:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 504:101: **E501** *line too long (105 > 100 characters)*

`scripts/fetch_jenkins_artifacts.py`:
- :red_circle: Line 28:9: **F841** *local variable 'result' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/utils/ssh.py (hunk 1 above)
Comment (mshriver) [reply cfme/utils/ssh.py]: @RonnyPfannschmidt Unless you feel strongly about it, I think I'd rather leave `rc` as-is, and not change everything to `return_code` as I think `rc` is descriptive enough in the context to correctly expect/understand what it is.

**Comment by dajoRH** [comment]
Lint report for commit ffe83edcc785b7ddb8be78248bf8e48dd5d3a479:

`scripts/fetch_jenkins_artifacts.py`:
- :red_circle: Line 28:9: **F841** *local variable 'result' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: scripts/create_cu_vm.py [hunk 2]
```diff
@@ -21,8 +21,8 @@
 
 
 def command_run(ssh_client, command, message):
-    exit_status, output = ssh_client.run_command(command)
-    if exit_status != 0:
+    result = ssh_client.run_command(command)
+    if result.failed0:
```
Comment (rlbabyuk) [inline scripts/create_cu_vm.py]: need a correction here

Comment (mshriver) [reply scripts/create_cu_vm.py]: done

File: scripts/create_cu_vm.py [hunk 3]
```diff
@@ -46,25 +46,25 @@ def config_cu_vm(ssh_client):
     logger.info('Setting up cron jobs on the CU VM')
 
     command_run(ssh_client, "cd /etc/init.d; wget {}/cu-disk-script.sh".format(url),
-        "CU: There was an error downloading disk script file")
+                "CU: There was an error downloading disk script file")
```
Comment (mshriver) [inline scripts/create_cu_vm.py]: Cleaning up hanging indent spacing.

**Pull request merged**